### DISCUSSION TITLE
Files are not deleted from IOS and keeps sending back

### DISCUSSION THREAD

**Discussion by gerroon** [vrtmrz/obsidian-livesync#348] [Q&A]
I am having issue with IOS not respecting the sync state\

- Delete 50 images on the desktop
- Sync Desktop
- Sync on IOS
- Images are not deleted

If I mess with th sync settings like verify and repair or  perform those files keep coming back I do not understand :( Even when rebuild the local database keeps those image files there. Weird.

**Comment by gerroon** [comment]
It turns out that it was a lowercase and uppercase collusion :( Damn Windows.

**Reply by vrtmrz** [reply]
Thank you for the important follow-up report!
I think we should do something about this. However, I still do not have any idea how to handle them...
